### DISCUSSION TITLE
怎么就这么喜欢画蛇添足

### DISCUSSION THREAD

**Discussion by MoonBreaker123** [MaaAssistantArknights/MaaAssistantArknights#10073] [General]
![image](https://github.com/user-attachments/assets/a0c295b6-73c0-4847-8cd5-006a270e6582)
没理解为什么要加改完成后选项，之前的一级菜单不好吗？改成二级不是画蛇添足？

**Comment by ABA2396** [comment]
🤓☝️我有个绝妙的点子，比如在下拉列表里塞20个选项

**Reply by MoonBreaker123** [reply]
好吧
